app/collins/solr/SolrConfig.scala

```diff
@@ -32,8 +32,9 @@ object SolrConfig extends Configurable {
   def socketTimeout = getInt("socketTimeout", 1000)
   def connectionTimeout = getInt("connectionTimeout", 5000)
   def maxTotalConnections = getInt("maxTotalConnections", 100)
+  def commitWithin = getInt("commitWithinMs", 50)
```

Comment:
yep, its discussed in bullet 3 in the description of the PR, but also https://lucene.apache.org/solr/5_2_0/solr-solrj/org/apache/solr/client/solrj/SolrClient.html#add(java.util.Collection, int)